packages/synapse-react-client/test/containers/AccessRequirementList/DataAccessRequestAccessorsFilesForm.test.tsx

```diff
@@ -262,7 +270,7 @@ describe('RequestDataAccessStep2: basic functionality', () => {
 
     await waitFor(() => {
       expect(mockUpdateDataAccessRequest).toHaveBeenCalled()
-      const updatedRequest = mockUpdateDataAccessRequest.mock.calls[0][0]
+      const updatedRequest = mockUpdateDataAccessRequest.mock.lastCall[0]
```

Comment:
and since we're now issuing multiple calls to mockUpdateDataAccessRequest, check the `lastCall`, instead of the first/only call